NOTICE.md

```diff
@@ -1,189 +1,34 @@
-|rand_chacha|0.2.2|MIT OR Apache-2.0|
-|rand_chacha|0.3.1|MIT OR Apache-2.0|
-|rand_core|0.5.1|MIT OR Apache-2.0|
-|rand_core|0.6.3|MIT OR Apache-2.0|
-|rand_hc|0.2.0|Apache 2.0<br/>MIT|
-|rand_pcg|0.2.1|MIT OR Apache-2.0|
-|redox_syscall|0.1.57|MIT|
-|redox_syscall|0.2.10|MIT|
-|redox_users|0.3.5|MIT|
-|redox_users|0.4.0|MIT|
-|regex|1.5.4|MIT OR Apache-2.0|
-|regex-automata|0.1.10|MIT<br/>Unlicense|
-|regex-syntax|0.6.25|Apache 2.0<br/>MIT|
-|remove_dir_all|0.5.3|Apache 2.0<br/>MIT|
-|rust-argon2|0.8.3|Apache 2.0<br/>MIT|
-|rustversion|1.0.6|MIT OR Apache-2.0|
-|ryu|1.0.9|Apache-2.0 OR BSL-1.0|
-|serde|1.0.137|MIT OR Apache-2.0|
-|serde_json|1.0.81|MIT OR Apache-2.0|
-|sharded-slab|0.1.4|MIT|
-|signal-hook-registry|1.4.0|Apache 2.0<br/>MIT|
-|siphasher|0.3.9|Apache 2.0<br/>MIT|
-|slab|0.4.5|MIT|
-|smallstr|0.2.0|Apache 2.0<br/>MIT|
-|smallvec|1.8.0|Apache 2.0<br/>MIT|
-|socket2|0.4.4|MIT OR Apache-2.0|
-|sqlparser|0.16.0|Apache 2.0|
-|syn|1.0.86|MIT OR Apache-2.0|
-|tempfile|3.3.0|MIT OR Apache-2.0|
-|terminfo|0.7.3|WTFPL|
-|textwrap|0.15.0|MIT|
-|thiserror|1.0.30|MIT OR Apache-2.0|
-|thiserror-impl|1.0.30|MIT OR Apache-2.0|
-|thread_local|1.1.4|Apache 2.0<br/>MIT|
-|time|0.3.9|MIT OR Apache-2.0|
-|tinyvec|1.5.1|Zlib OR Apache-2.0 OR MIT|
-|tinyvec_macros|0.1.0|MIT OR Apache-2.0 OR Zlib|
-|tokio|1.17.0|MIT|
-|tokio-io-timeout|1.2.0|Apache 2.0<br/>MIT|
-|tokio-macros|1.7.0|MIT|
-|tokio-stream|0.1.8|MIT|
-|tokio-util|0.6.9|MIT|
-|tokio-util|0.7.0|MIT|
-|tonic|0.6.2|MIT|
-|tonic-build|0.6.2|MIT|
-|tower|0.4.12|MIT|
-|tower-layer|0.3.1|MIT|
-|tower-service|0.3.1|MIT|
-|tracing|0.1.34|MIT|
-|tracing-attributes|0.1.20|MIT|
-|tracing-core|0.1.22|MIT|
-|tracing-futures|0.2.5|MIT|
-|tracing-subscriber|0.3.11|MIT|
-|try-lock|0.2.3|MIT|
-|unicode-bidi|0.3.7|Apache-2.0<br/>MIT|
-|unicode-normalization|0.1.19|Apache 2.0<br/>MIT|
-|unicode-segmentation|1.9.0|Apache 2.0<br/>MIT|
-|unicode-width|0.1.9|Apache 2.0<br/>MIT|
-|unicode-xid|0.2.2|MIT OR Apache-2.0|
-|url|2.2.2|Apache 2.0<br/>MIT|
-|valuable|0.1.0|MIT|
-|vcpkg|0.2.15|Apache 2.0<br/>MIT|
-|databroker-api|0.17.0|Apache 2.0|
-|databroker|0.17.0|Apache 2.0|
-|databroker-cli|0.17.0|Apache 2.0|
-|databroker-examples|0.17.0|Apache 2.0|
-|vergen|7.0.0|MIT OR Apache-2.0|
-|version_check|0.9.4|Apache 2.0<br/>MIT|
-|void|1.0.2|MIT|
-|want|0.3.0|MIT|
-|wasi|0.10.2+wasi-snapshot-preview1|Apache-2.0 WITH LLVM-exception OR Apache-2.0 OR MIT|
-|wasi|0.9.0+wasi-snapshot-preview1|Apache-2.0 WITH LLVM-exception OR Apache-2.0 OR MIT|
-|which|4.2.4|MIT|
-|winapi|0.3.9|Apache 2.0<br/>MIT|
-|winapi-i686-pc-windows-gnu|0.4.0|Apache 2.0<br/>MIT|
-|winapi-x86_64-pc-windows-gnu|0.4.0|Apache 2.0<br/>MIT|
+During build thirdparty components may be included. Databroker is a Rust
+project, all our dependencies are declared in the project's Cargo files.
+Depending on the build date and your build environment different transitive
+dependecies might be included. Artifacts published in this repo contain a list
```

Comment:
There are some minor spelling errors on this line and next line that we better shall fix